site/en/blog/don-t-build-blobs-construct-them/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Don't build blobs, construct them
+description: >
+  Recent spec changes to the File API include a new constructor for Blob, which essentially makes BlobBuilder irrelevant.
+authors:
+  - ericbidelman
+date: 2012-06-10
+updated: 2019-02-09
+
+---
+
+Here's a heads up to all you [BlobBuilder](https://developer.mozilla.org/docs/Web/API/BlobBuilder) fans...it's going bye bye!
+
+`BlobBuilder` is a handy API for creating Blobs (or Files) in JavaScript. It's been around since Chrome 8, FF 6, and IE 10 but has never shipped in Safari,...and likely never will. Recent spec changes to the [File API](https://w3c.github.io/FileAPI/#dfn-Blob) include a new constructor for `Blob`, which essentially makes `BlobBuilder` irrelevant. In fact, Safari nightlies have already disabled it and Chrome will start to warn you in the console very soon.
```

Comment:
```suggestion
Here's a heads up to all you [BlobBuilder](https://developer.mozilla.org/docs/Web/API/BlobBuilder) fans, it's going bye bye!

`BlobBuilder` is a handy API for creating Blobs (or Files) in JavaScript. It's been around since Chrome 8, FF 6, and IE 10 but has never shipped in Safari, and likely never will. Recent spec changes to the [File API](https://w3c.github.io/FileAPI/#dfn-Blob) include a new constructor for `Blob`, which essentially makes `BlobBuilder` irrelevant. In fact, Safari nightlies have already disabled it and Chrome will start to warn you in the console very soon.
```